CHANGES.rst

```diff
@@ -4,6 +4,8 @@ Products.CMFCore Changelog
 2.4.7 (unreleased)
 ------------------
 
+- Change the default value of uid to None on CMFCatalogAware's reindexObject method (`#80 <https://github.com/zopefoundation/Products.CMFCore/issue`_)
+
```

Comment:
Can you fix the issue id? You mention 80, but it should be 87, also in the title of this PR.
And the url should then end with `issues/87`.

Reply:
woops, done